### PR TITLE
Add some resiliency to the scripts

### PR DESCRIPTION
**Pull request by rsnyman**
- Handle missing files better
- Output errors and warnings to stderr so that they don't interfere with output

### CODE REVIEW

File: ibutsu_utils/xmerge.py [hunk 1]
```diff
@@ -32,6 +37,8 @@ def merge_files(filenames):
 def main():
     args = parse_args()
     output_xml = merge_files(args.input)
+    if not output_xml:
+        return
```
Comment (john-dupuy) [inline ibutsu_utils/xmerge.py:41]: Do we also need a warning message here?

Comment (rsnyman) [reply ibutsu_utils/xmerge.py:41]: Generally you're only going to hit that point if there are not files. I used to have a message there, but then I got both the message about the missing file and the message about no output, so I figured it was a little overboard.

Comment (rsnyman) [reply ibutsu_utils/xmerge.py:41]:
```
$ ibutsu-xmerge doesnotexist.xml
WARNING: doesnotexist.xml does not exist
WARNING: No files provided
```

Comment (john-dupuy) [reply ibutsu_utils/xmerge.py:41]: Aha! That makes sense.

**Review by john-dupuy — APPROVED** [review]

**Pull request merged**